portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,6 +41,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Does this fulfill the tODO at the top of the class?

Reply:
I'm not sure... I was going to wait until I had finished all of the steps of the walkthrough to get rid of it. I have one left to go.

Reply:
sounds good to me (SGTM)